assets/src/edit-story/components/form/color/test/color.js

```diff
@@ -31,10 +31,10 @@ jest.mock('../applyOpacityChange', () => jest.fn());
 
 function arrange(props = {}) {
   const onChange = jest.fn();
-  const { queryByLabelText } = renderWithTheme(
+  const { queryByLabelText, getByRole } = renderWithTheme(
     <Color label="Color" onChange={onChange} {...props} />
   );
-  const colorPreview = queryByLabelText(/Edit: Color/);
+  const colorPreview = getByRole('button');
```

Comment:
Nit: You can pass {name: 'Color'} to disambiguate with other buttons that may be added in the future.

Reply:
Good call, done!